lib/amazon/gateway.rb

```diff
@@ -13,6 +13,10 @@ def initialize(config)
       @communicator_strategy = config[:communicator_strategy]
     end
 
+    def put_object_acl(bucket_name, object_key, acl)
```

Comment:
Does this belong on the gateway or just on the bucket?

Reply:
It could just be on the bucket, I did it this way because it made it easier to test (the Bucket class has no tests). Also it makes it easy to use the fake communicator strategy in the MCM, since this will be used as part of enqueuing a daily job and I don't want staging accidentally making a real call.

Reply:
The way this gem is structured it should probably be only on the bucket.

Reply:
Ok, will do